index.js

```diff
@@ -1,11 +1,47 @@
 class SortedList {
-  constructor() {}
-  add(item) {}
-  get(pos) {}
-  max() {}
-  min() {}
-  average() {}
-  sum() {}
-};
+  constructor() {
+    this.items = [];
+    this.length = 0;
+  }
+
+  add(item) {
+    this.items.push(item);
+    this.length = this.items.length;
+
+    this.items.sort((a, b) => a - b);
+  }
+
+  get(pos) {
+    return this.items[pos - 1];
+  }
+
+  max() {
+    this.checkLength();
```

Comment:
Molt be... un poco vago pero si seño DRY!!!

Reply:
y que de la sensación de "un poco vago" es bueno o malo? 😅 pensé que hacerlo así era mejor